src/notes.ts

```diff
@@ -168,15 +173,20 @@ chrome.storage.onChanged.addListener((changes, areaName) => {
 
       // This should not happen (fallback)
       if (!state.active) {
+        const newActive = Object.keys(newNotes).sort()[0];
+        if (newActive) {
+          state.active = newActive;
+        }
         return;
       }
 
       // Reactivate edited note in other tabs only
+      // Reactive clipboard if edited from background using Context menu
       if (
         (state.active in oldNotes) &&
         (state.active in newNotes) &&
         (newNotes[state.active].content !== oldNotes[state.active].content) &&
-        (localStorage.getItem("notesChangedBy") !== tabId)
+        ((localStorage.getItem("notesChangedBy") !== tabId) || changes["clipboard"])
```

Comment:
If "clipboard" was changed from the background, re-activate it even if in current window, to see the new content. This is true on a initial clipboard set only. After that, we need to rely on "notesChangedBy" to be NOT set, as the change is coming NOT from the user, but from the background, so we want to show the received content automatically.